dmc_configs/dmc_configs/README.md

```diff
@@ -0,0 +1,176 @@
+
+The main output measure is accuracy and rotation error as a function of training rotations.
+
+**Notes:**
+- Training rotations should be structured:
+  1. First 6 rotations = cube faces
+  2. Next 8 rotations = cube corners
+  3. Remaining = random rotations (as otherwise introduces redundancy)
+
+**TODO:**
+- Configs need to be specified
+- Comparable configs needed for ViT model comparison
+
+## Figure 7: Computationally Efficient Learning and Inference
+
+Consists of 8 experiments:
+
+### Inference (7 experiments):
+- `dist_agent_1lm_randrot_nohyp_x_percent_5p` - 5% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_10p` - 10% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_20p` - 20% threshold (default for other experiments)
+- `dist_agent_1lm_randrot_nohyp_x_percent_40p` - 40% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_80p` - 80% threshold
+- `dist_agent_1lm_randrot_nohyp_x_percent_100p` - 100% threshold
+
+### Training (1 experiment):
+- `dist_agent_77obj_1rot_trained`*
+
+This means performance is evaluated with:
+- 77 objects
+- 14 random rotations
+- No sensor noise**
+- No hypothesis-testing**
+- No voting
+
+*Single rotation evaluation due to FLOPs counting overhead, so we will extrapolate total FLOPs to 14 rotations based on this/
+or can compare to a ViT trained on 1 rotation.
+
+**Due to ViT model comparison.
+
+
+The main output measure is accuracy and FLOPs as a function of x-percent threshold.
+
+**TODO:**
+- Configs need specification (including `dist_agent_77obj_1rot_trained` in `dmc_pretrain_experiments.py`)
+- Comparable config needed to generate views corresponding to the 14 random, evaluated rotations, which can then be passed to the ViT model(s) for comparison.
+
+## Figure 8: Multi-Modal Transfer
+
+Consists of 4 experiments:
+- `dist_agent_1lm_randrot_noise` - "Standard" Monty ("dist_on_dist")
+- `dist_on_touch_1lm_randrot_noise` - "dist_on_touch"
+- `touch_agent_1lm_randrot_noise` - "touch_on_touch" baseline
+- `touch_on_dist_1lm_randrot_noise` - "touch_on_dist"
+
+This means performance is evaluated with:
+- 77 objects
+- 14 random rotations
+- Sensor noise
+- Hypothesis-testing policy active
+- No voting
+
+The main output measure is accuracy and rotation error for within/across modality inference.
+
+## Figure 9: Structured Object Representations
+
+Consists of 1 experiment:
+- `dist_agent_1lm_randrot_noise_10simobj`
+
+This means performance is evaluated with:
+- 10 morphologically similar objects
+  - This config should therefore also use the model trained on the 10 morphologically similar objects.
+- Random rotations
+- Sensor noise
+- Hypothesis-testing policy active
+- No voting
+
+The main output measure is a dendrogram showing evidence score clustering for the 10 objects.
+
+**TODO:**
+- Config needs specification, including training config for similar objects in `dmc_pretrain_experiments.py`.
```

Comment:
Also will need to run this experiment with SELECTIVE logging on so we get the evidence values to analyze. We could also use the standard 77 object pretrained models and then just create the dendrogram for a subset of the objects for clarity.